classes/class-p4-loader.php

```diff
@@ -201,4 +201,18 @@ public function do_not_delete_autosave( $delete = null, $post = null, $force_del
 
 		return false;
 	}
+
+	/**
+	 * @param string $rel_path Relative path to the file.
+	 * @return int timestamp of file creation
+	 */
+	public static function theme_file_ver( string $rel_path ): int {
+		$filepath = trailingslashit( get_template_directory() ) . $rel_path;
+		$ctime = filectime( $filepath );
```

Comment:
> Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

:grin:

Reply:
_starts to write a 50 pages ADR to explain how wrong this is 🔨_